Test.Microsoft.NET.Build.Containers.Filesystem/TargetsTests.cs

```diff
@@ -46,33 +46,38 @@ private Project InitProject(Dictionary<string, string> bonusProps)
         props["_TargetFrameworkVersionWithoutV"] = "7.0";
         props["_NativeExecutableExtension"] = ".exe"; //TODO: windows/unix split here
         props["Version"] = "1.0.0"; // TODO: need to test non-compliant version strings here
+        props["NETCoreSdkVersion"] = "7.0.100"; // we manipulate this value during evaluation, so we need a sane default.
```

Comment:
```suggestion
        props["NETCoreSdkVersion"] = "7.0.100"; // we manipulate this value during evaluation, so we need a good default.
```

(this would trigger PoliCheck and it bothers some people)